src/core/client/systems/interaction.ts

```diff
@@ -230,15 +231,28 @@ export class InteractionController {
 
             // Object Type
             if (closestTarget.type === 'object') {
-                const hash = native.getEntityModel(closestTarget.scriptID);
+                const hash = native.getEntityModel(closestTarget.scriptID);                
                 const isValid = ObjectWheelMenu.isModelValidObject(hash);
                 if (isValid) {
+                    const targetCoords = native.getEntityCoords(closestTarget.scriptID, false);
+                    let item = null;
+                    for (const closesItem of closestItems) {
```

Comment:
this should be closestItemtItem not closesItem

Reply:
closestItem*

Reply:
yes, will fix it